cognite_toolkit/default.packages.yaml

```diff
@@ -18,3 +18,8 @@ packages:
     - cdf_apm_base
     - cdf_infield_common
     - cdf_infield_location
+  example_pump:
+    - example_pump_asset_hierarchy
+    - cdf_asset_source_model
+    - example_pump_data_model
```

Comment:
There's something that is not logical about the naming of the modules. You would think that there is a dependency on the data model from the example_pump_asset_hierarchy. I think we need more info in the READMEs to also describe the relationships between the models. Maybe we should introduce a naming convention for modules with data? like example_pump_asset_hierarchy_data